pallets/runtime/tests/src/sto_test.rs

```diff
@@ -270,13 +270,6 @@ fn raise_unhappy_path() {
 
     let correct_venue = create_venue(alice_signed.clone(), VenueType::Sto);
 
-    // Raise asset not created
-    check_fundraiser(
-        default_tiers.clone(),
-        correct_venue,
-        Error::InvalidPortfolio.into(),
-    );
```

Comment:
Can you elaborate on this diff?

Reply:
This was an uneeded check. The code was checking that the portfolio has some existing balance but we don't need that to be true. New/empty portfolios are also fine.